docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_CRNN_From_Tensorflow.md

```diff
@@ -3,54 +3,50 @@
 This tutorial explains how to convert a CRNN model to Intermediate Representation (IR).
 
 There are several public versions of TensorFlow CRNN model implementation available on GitHub. This tutorial explains how to convert the model from
-the [CRNN Tensorflow](https://github.com/MaybeShewill-CV/CRNN_Tensorflow) repository to IR. 
+the [CRNN Tensorflow](https://github.com/MaybeShewill-CV/CRNN_Tensorflow) repository to IR, which was validated with python 3.7, tensorflow 1.15.0, protobuf 3.19.0.
 If you have another implementation of CRNN model, it can be converted to OpenVINO IR in a similar way. You need to get inference graph and run Model Optimizer on it.
 
 **To convert this model to the IR:**
 
 **Step 1.** Clone this GitHub repository and checkout the commit:
     1. Clone repository:
 ```sh
- git clone https://github.com/MaybeShewill-CV/CRNN_Tensorflow.git
+git clone https://github.com/MaybeShewill-CV/CRNN_Tensorflow.git
 ```
-    2. Checkout necessary commit:
+    2. Go to the `CRNN_Tensorflow` directory of the cloned repository:
+```sh
+cd path/to/CRNN_Tensorflow
+```
+    3. Checkout necessary commit:
 ```sh
 git checkout 64f1f1867bffaacfeacc7a80eebf5834a5726122
 ```
 
 **Step 2.** Train the model, using framework or use the pretrained checkpoint provided in this repository.
 
 **Step 3.** Create an inference graph:
-    1. Go to the `CRNN_Tensorflow` directory of the cloned repository:
-```sh
-cd path/to/CRNN_Tensorflow
-```
-    2. Add `CRNN_Tensorflow` folder to `PYTHONPATH`.
+    1. Add `CRNN_Tensorflow` folder to `PYTHONPATH`.
        * For Linux OS:
 ```sh
 export PYTHONPATH="${PYTHONPATH}:/path/to/CRNN_Tensorflow/"
 ```
        * For  Windows OS add `/path/to/CRNN_Tensorflow/` to the `PYTHONPATH` environment variable in settings.
-    3. Open the `tools/test_shadownet.py` script. After `saver.restore(sess=sess, save_path=weights_path)` line, add the following code:
+    2. Open the `tools/demo_shadownet.py` script. After `saver.restore(sess=sess, save_path=weights_path)` line, add the following code:
 ```python
-import tensorflow as tf
 from tensorflow.python.framework import graph_io
-frozen = tf.compat.v1.graph_util.convert_variables_to_constants(sess, sess.graph_def, ['shadow/LSTMLayers/transpose_time_major'])
+frozen = tf.graph_util.convert_variables_to_constants(sess, sess.graph_def, ['shadow/LSTMLayers/transpose_time_major'])
```

Comment:
is it a reason why we request users to have `tensorflow 1.15.0`? In TF2, the freezing should work. I can't see that here we use something from `contrib`.
Please check

Reply:
@rkazants, thanks for your review. Actually original repo use `rnn` module from `tensorflow.contrib` to build crnn model sturcture, which is required to freeze TF model: 
- `from crnn_model import crnn_model`:https://github.com/MaybeShewill-CV/CRNN_Tensorflow/blob/64f1f1867bffaacfeacc7a80eebf5834a5726122/tools/demo_shadownet.py#L26
- `from tensorflow.contrib import rnn
`:https://github.com/MaybeShewill-CV/CRNN_Tensorflow/blob/64f1f1867bffaacfeacc7a80eebf5834a5726122/crnn_model/crnn_model.py#L14
However, `contrib` module is drop in TF2 without compatible API.